lib/Doctrine/DBAL/Migrations/Tools/Console/Command/MigrateCommand.php

```diff
@@ -153,4 +139,51 @@ public function execute(InputInterface $input, OutputInterface $output)
             }
         }
     }
+
+    /**
+     * @param $question
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return bool
+     */
+    private function getConfirmationToExecute($question, InputInterface $input, OutputInterface $output)
```

Comment:
I like  method names like `canExecute` for stuff like this. Just reads a bit cleaner in use.

```php
if (! $this->canExecute($question, $input, $output)) {
  // ...
}
```